contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -7858,12 +7858,22 @@ pltsql_param_fetch(ParamListInfo params,
 		}
 	}
 
+	if (saved_expr_kind == EXPRKIND_TARGET)
+	{
+		/* Let extension to set value of param dynamically during execution when variables appears in TargetList */
+		prm->pflags = 0;
+	}
+	else
+	{
+		/* We can always mark params as "const" for executor's purposes */
```

Comment:
Update comment "In other cases we can mark params as const"

Reply:
Added comment